build-tools/packages/build-cli/docs/vnext.md

```diff
@@ -0,0 +1,32 @@
+`flub vnext`
+============
+
+Vnext commands are new implementations of standard flub commands using new infrastructure.
+
+* [`flub vnext check latestVersions VERSION RELEASE_GROUP`](#flub-vnext-check-latestversions-version-release_group)
```

Comment:
I suggest marking the existing command as deprecated now. That will just add a warning message to the output, but it will continue working. Makes it clear that it needs to be removed at some point.